public/js/index.js

```diff
@@ -758,6 +799,64 @@ Index.migrateProgress = function () {
     }
 };
 
+/**
+ * Restore and update column width, data store in localstorge.
+ */
+Index.resizableColumns = function () {
+    let currentHeader;
+    let startX;
+    let startWidth;
+
+    const headers = document.querySelectorAll("#header-row th");
+    headers.forEach((header, index) => {
+        // restore Column Width
+        const savedWidth = localStorage.getItem(`resizeColumn${index}`);
+        if (savedWidth) {
+            header.style.width = savedWidth;
+        }
+        // init
+        header.addEventListener('mousedown', function (event) {
+            if (event.offsetX > header.offsetWidth - 10) {
+                currentHeader = header;
+                startX = event.clientX;
+                startWidth = header.offsetWidth;
+
+                document.addEventListener('mousemove', resizeColumn);
+                document.addEventListener('mouseup', stopResize);
+
+                document.body.style.cursor = 'col-resize';
+            }
+        });
+        header.addEventListener('mousemove', function (event) {
+            if (event.offsetX > header.offsetWidth - 10) {
+                header.style.cursor = 'col-resize';
+            } else {
+                header.style.cursor = 'default';
+            }
+        });
+    });
+
+    function resizeColumn(event) {
+        if (currentHeader) {
+            const newWidth = startWidth + (event.clientX - startX);
+            if (newWidth > 0) {
+                currentHeader.style.width = newWidth + 'px';
+            }
+        }
+    }
+
+    function stopResize() {
+        if (currentHeader) {
+            const index = Array.from(headers).indexOf(currentHeader);
+            localStorage.setItem(`resizeColumn${index}`, currentHeader.style.width);
+            currentHeader = null;
+        }
+        document.removeEventListener('mousemove', resizeColumn);
```

Comment:
I feel like this function needs a way to block the sort-on-click behavior that datatables has;  

Currently if I drag to resize a column and release the mouse while in the column, it'll trigger a sort, which is very disorienting and resets the column width.  

https://github.com/user-attachments/assets/f11583b7-e663-4e78-950d-5631c832d556